packages/app-core/src/ui/App/DrawerHeader.tsx

```diff
@@ -39,21 +39,38 @@ const useStyles = makeStyles()(theme => ({
   header: {
     background: theme.palette.secondary.main,
   },
+  headerFocused: {
+    background: theme.palette.secondary.light,
+  },
+  headerUnfocused: {
+    background: theme.palette.secondary.dark,
+  },
 }))
 
 export default observer(function ({
   session,
   setToolbarHeight,
 }: {
-  session: SessionWithDrawerWidgets
+  session: SessionWithFocusedViewAndDrawerWidgets
   setToolbarHeight: (arg: number) => void
 }) {
   const { classes } = useStyles()
+  const focusedViewId = session.focusedViewId
+  // @ts-ignore
+  const viewWidgetId = session.visibleWidget?.view?.id
+  const isFocused = focusedViewId && focusedViewId === viewWidgetId
 
   return (
     <AppBar
       position="sticky"
-      className={classes.header}
+      elevation={0}
```

Comment:
addition of elevation 0?

Reply:
good catch, accidental leave in, but that isn't causing the snap changes